requirements.txt

```diff
@@ -1,4 +1,4 @@
-cryptography==1.8.2
+cryptography>=1.8.1,<1.10
```

Comment:
in ``vapid`` we are using ``cryptography>=1.8.2,<=1.10``. So this should not be same?